CHANGELOG-3.4.md

```diff
@@ -726,4 +726,6 @@ To get the diff between two versions, go to https://github.com/symfony/symfony/c
  * feature #22803 [DI] Deprecate Container::initialized() for privates (ro0NL)
  * feature #22828 [Finder] Deprecate FilterIterator (ogizanagi)
  * feature #22826 [Validator] improve strict option value deprecation (xabbuh)
+ * feature #16835 [Security] Deprecated `GuardAuthenticatorInterface` in favor of `AuthenticatorInterface` (Amo)
+ * feature #16835 [Security] When extending `AbstractGuardAuthenticator` it's no longer allowed to return `null` from `getCredentials()` (Amo)
```

Comment:
I really meant `src/Symfony/Component/Security/CHANGELOG.md`
this one is autogenerated so this should be reverted :)